examples/rust/components/dog-fetcher/wit/deps/wasi-cli-0.2.2/package.wit

```diff
@@ -1,19 +1,19 @@
-package wasi:cli@0.2.1;
+package wasi:cli@0.2.2;
```

Comment:
looks like we should be able to just delete `wit` directory now that we don't generate bindings in this example?

Reply:
Looks like you can't just delete it outright, but I'll file a bug